### PR TITLE
READY : Fix routine sessionsRun ( empty sessions )

### PR DESCRIPTION
**Pull request by dmvict**
Routine should accept some sessions to take resource from it, otherwise, it creates procedure which wait for infinite time

### CODE REVIEW

**Comment by dmvict** [comment]
Should I write the full test routine?
```
/* qqq for Yevhen : cover please */
function sessionsRun_body( o )
```

**Comment by Wandalen** [comment]
Thanks. Fix?

**Comment by dmvict** [comment]
> Thanks. Fix?

https://github.com/Wandalen/wConsequence/pull/99/files#diff-f55417198cde334fe84972e3f700712de8530703a015f0b258724581ca958840R280

File: proto/wtools/abase/l9.test/consequence/Process.test.s [hunk 1]
```diff
@@ -237,6 +237,40 @@ function ready( test )
...
+    onRun: ( session ) => { return session },
+    ready: null,
+    readyName: 'ready',
+    sessions : [],
+    ready : null,
+  };
+  var got = _.sessionsRun( o );
+  test.true( got === o );
+  test.true( _.consequenceIs( got.ready ) );
+}
```
Comment (Wandalen) [inline proto/wtools/abase/l9.test/consequence/Process.test.s:273]: ?

Comment (dmvict) [reply proto/wtools/abase/l9.test/consequence/Process.test.s:273]: Do not change interface of routine 

```js
  return o;
```

**Pull request merged**